staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -220,7 +220,7 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 				return
 			}
 
-			glog.Warningf("rejected by webhook %v %t: %v", hook.Name, err, err)
+			glog.Warningf("rejected by webhook %q of type %T: %v", hook.Name, err, err)
```

Comment:
The wording seems wrong. The type is the type of the error, not the webhook type. 

Anyway, do you know why we print the error type in the first place?  Can we just print the error as `%#v`?

Reply:
I believe it's just for debugging. `%#v` seems more straightforward.